### PR TITLE
feat(Datatable): add first version of public API

### PR DESCRIPTION
**Pull request by ajkl2533**
Ref FEP-1594

### CODE REVIEW

File: src/components/Datatable/Datatable.stories.tsx [hunk 1]
```diff
@@ -16,12 +32,179 @@ export default {
...
+      },
+    }: LinkCellRendererProps<string> & {
+      row: Row<Data>;
+    }): React.ReactElement => {
+      dayjs.extend(relativeTime);
+
+      return (
+        <TextCell
+          column={column}
+          formater={(val) => dayjs(val).to(dayjs(lastObservationDate), true)}
```
Comment (vaclav-zeman) [inline src/components/Datatable/Datatable.stories.tsx:156]: `formater` - missing `t`

**Review by vaclav-zeman — APPROVED** [review]
LGTM!

File: src/components/Datatable/Table/Table.types.ts [hunk 2]
```diff
@@ -1,21 +1,68 @@
...
+  label: string;
+  name: string;
+  onClick?: (rowId: number | string, row: D) => void;
+};
+type HandlerRowAction<D> = Required<BaseRowAction<D>>;
+type AbsolutLinkRowAction<D> = BaseRowAction<D> & {
+  hrefComposer: (rowId: number | string, row: D) => string;
+  toComposer?: never;
+};
+type RelativeLinkrRowAction<D> = BaseRowAction<D> & {
```
Comment (PrincAm) [inline src/components/Datatable/Table/Table.types.ts:16]: typo

**Review by PrincAm — APPROVED** [review]
GJ!

**Pull request merged**